libraries/chain/protocol/transaction.cpp

```diff
@@ -334,7 +334,7 @@ set<public_key_type> signed_transaction::get_required_signatures(
    for( auto& owner : required_owner )
       s.check_authority( get_owner( owner ) );
    for( auto& active : required_active )
-      s.check_authority( active  );
+      s.check_authority( active ) || s.check_authority( get_owner( active ) );
```

Comment:
Perhaps better check `owner` first, so when the client can sign with owner key, no need to sign with active key at same time? (See #197)